frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -80,6 +80,12 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
     highlightedIds: selectedIds,
   };
 
+  const closeSidebarAndHandleReset = React.useCallback(() => {
+    setSelectedTask(null);
+    selectedTaskRef.current = null;
```

Comment:
Not a fan of this solution. It looks _wrong_ but it seems to be a fault of the way the Pipeline Builder was written by me back in 4.4. I think this is an _okay_ solution to back port. In 4.6, I'll fix it another way in #4805 when I rebase it and clean it up.